tests/FakeGRPC.Dockerfile

```diff
@@ -0,0 +1,18 @@
+FROM node:14-slim
+
+WORKDIR /app
+
+COPY .npmrc .
+
+COPY fakeGRPCServer/package.json .
+COPY fakeGRPCServer/yarn.lock .
+
+RUN yarn
+
+COPY fakeGRPCServer .
```

Comment:
did you find out why do we copy that way?

Reply:
@Barmil if we copy the entire codebase before installing our dependencies, on each change we make during development Docker will have to reinstall all our dependencies, even though most of the time they haven’t changed